lib/vtls/schannel_verify.c

```diff
@@ -77,21 +77,150 @@ static int is_cr_or_lf(char c)
   return c == '\r' || c == '\n';
 }
 
-static CURLcode add_certs_to_store(HCERTSTORE trust_store,
-                                   const char *ca_file,
-                                   struct Curl_easy *data)
+static const char *c_memstr(const char *str, const char *str_limit,
+                            const char *sub, size_t size_sub)
+{
+  const char *p;
+  char first;
+  if((str + size_sub) > str_limit)
+    return NULL;
+  first = *sub;
+  for(p = str; p <= (str_limit - size_sub); p++)
+    if(((*p) == first) && (memcmp(p, sub, size_sub) == 0))
+      return p;
+
+  return NULL;
+}
+
+static CURLcode add_certs_data_to_store(HCERTSTORE trust_store,
+                                        const char *ca_buffer,
+                                        size_t ca_buffer_size,
+                                        const char *ca_file_text,
+                                        struct Curl_easy *data)
+{
+  size_t BEGIN_CERT_LEN = strlen(BEGIN_CERT);
+  size_t END_CERT_LEN = strlen(END_CERT);
```

Comment:
We reserve all-uppercase names for defines and not variables. Should these also be set 'const' ?

Reply:
done